Catalogs/Vehicles/VehicleCatalog.xosc

```diff
@@ -167,7 +163,7 @@
       <Performance maxSpeed="70" maxDeceleration="10" maxAcceleration="10" />
       <Axles>
         <!-- maxSteering is expected in degrees by openPASS although the standard defines rad. This will be corrected in future. Esmini is not affected by the change. -->
```

Comment:
Please also remove this comment

Reply:
I missed that one. Thanks for the hint! Fixed in c792d2a2075fb0afdc7acbdcfad7c933a31b4612